build-tools/packages/build-cli/package.json

```diff
@@ -145,29 +145,29 @@
 	"devDependencies": {
 		"@biomejs/biome": "~1.9.3",
 		"@fluidframework/build-common": "^2.0.3",
-		"@fluidframework/eslint-config-fluid": "^5.3.0",
+		"@fluidframework/eslint-config-fluid": "^5.4.0",
 		"@oclif/test": "^4.1.0",
-		"@types/async": "^3.2.20",
-		"@types/chai": "^4.3.5",
+		"@types/async": "^3.2.24",
+		"@types/chai": "^4.3.20",
 		"@types/chai-arrays": "^2.0.3",
-		"@types/debug": "^4.1.7",
+		"@types/debug": "^4.1.12",
 		"@types/fs-extra": "^11.0.4",
 		"@types/issue-parser": "^3.0.5",
 		"@types/mdast": "^4.0.4",
 		"@types/mocha": "^10.0.9",
-		"@types/node": "^18.18.6",
-		"@types/node-fetch": "^2.5.10",
+		"@types/node": "^18.19.59",
+		"@types/node-fetch": "^2.6.11",
```

Comment:
Doesn't look like build-cli uses node-fetch, might be able to remove the dep and the types altogether.

Reply:
Commented there, I don't see a build issue without it.